controllers/specialresource.go

```diff
@@ -56,7 +56,7 @@ func SpecialResourcesReconcile(ctx context.Context, r *SpecialResourceReconciler
 		}
 		parent, err := r.Storage.CheckConfigMapEntry(ctx, req.Name, obj)
 		if err != nil {
-			r.StatusUpdater.UpdateWithState(ctx, &r.parent, fmt.Sprintf("%v", err))
```

Comment:
don't we want to set the condition/ state here to error?

Reply:
This is a case when the SR was not found, so what would we update?
And I'm afraid that `r.parent` might just contain something from previous reconciliation.